docs/kyma/docs/036-inst-xip.io-domain.md

```diff
@@ -10,52 +10,75 @@ This solution is not suitable for a production environment but makes for a great
 
 ## Prerequisites
 
-The prerequisites match these listed in the **Install Kyma on a GKE cluster** document. However, you don't need to prepare a domain for your cluster as it is replaced by a wildcard DNS provided by [`xip.io`](http://xip.io/).
+The prerequisites match these listed in [this](#installation-install-kyma-on-a-gke-cluster) document. However, you don't need to prepare a domain for your cluster as it is replaced by a wildcard DNS provided by [`xip.io`](http://xip.io/).
```

Comment:
Isn't using "\[this\]\(link\) document" a bad practice? Maybe:
```suggestion
The prerequisites match these listed in [Install on GKE cluster document](#installation-install-kyma-on-a-gke-cluster). However, you don't need to prepare a domain for your cluster as it is replaced by a wildcard DNS provided by [`xip.io`](http://xip.io/).
```

Reply:
No, I (we) don't consider it a bad practice. Now that linking's available for us to use we'll switch all cross-doc refs to this format.